src/Services/SubscriptionService.php

```diff
@@ -95,23 +94,6 @@ public function update($subscriptionId, $data)
         return $this->client()->put($data, 'subscriptions/{subscriptionId}', ['subscriptionId' => $subscriptionId]);
     }
 
-    /**
-     * @param string $subscriptionId
-     * @param array|JsonSerializable|SubscriptionCancel $data
-     *
-     * @throws UnprocessableEntityException The input data does not valid
-     *
-     * @return Subscription
-     */
-    public function cancel($subscriptionId, $data)
```

Comment:
it will break merchant integration when they update SDK?

Reply:
Yes.